cdk/script/export-monitoring-file

```diff
@@ -19,7 +19,14 @@ then
 
 	echo "Zipping Built Files"
 	cd "${absolute_src_directory}" # Go to monitoring/dist folder
-	zip -FSr "${ROOT_DIR}/cdk/dist/cmp-monitoring-lambda.zip" "." # Zip the monitoring/dist folder into cdk/dist
+
+	newDir="${ROOT_DIR}/cdk/dist/"
+
+	if ! [[ -d "$newDir" ]] ; then
+		mkdir -p "$newDir"
+	fi
```

Comment:
The if statement can be removed here because `mkdir -p` fails silently if the directory already exists:

> with this option specified, no error will be reported if a directory given as an operand already exists.

Reply:
Thanks!